processor/src/advice/source.rs

```diff
@@ -3,22 +3,22 @@ use super::{Felt, Word};
 // ADVICE SOURCE
 // ================================================================================================
 
-/// Placeholder for advice provider stack mutation.
+/// Specifies the source the value(s) to be pushed onto the advice stack.
```

Comment:
nit: should this be "Specifies the source of the value(s)..."?

Reply:
yes - fixed!